src/YHttpHandler.cc

```diff
@@ -35,6 +38,9 @@ int YHttpHandler::handle(struct MHD_Connection* connection,
     if (!encoding.empty())
         MHD_add_response_header(response, MHD_HTTP_HEADER_CONTENT_ENCODING, encoding.c_str());
 
+    yuiMilestone() << "Sending response: code: " << errorCode() << ", body size: " << body_str.length()
```

Comment:
Maybe log this with `yuiDebug()`, not `yuiMilestone()` so it doesn't clutter the y2logs for every request and reply by default?

Reply:
Um, I'd keep `yuiMilestone` (at least for now). There should not be many incoming requests for the REST API, basically checking status of few widgets and then sending some user events.

If it turns out to be too verbose we can decrease it later.

The REST API is still quite a new so I'd rather prefer more logging, after stabilizing we could review the logging and decrease the less important ones.